dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+| `type`        | Optional | String                                | Used as a shorthand, e.g., `type: 'image'` implies required fields `image`, `title`, `sponsoredBy`, `clickUrl`.                   |
+| `title`       | Required | Object                                | The title of the ad, usually a call to action or a brand name.  For properties, see [`native.title`](#native-title).              |
+| `body`        | Required | Object                                | Text of the ad copy.  For properties, see [`native.body`](#native-body).                                                          |
+| `sponsoredBy` | Required | String                                | The name of the brand associated with the ad.  For properties, see [`native.sponsoredBy`](#native-sponsoredby).                   |
+| `icon`        | Optional | Object                                | The brand icon that will appear with the ad.  For properties, see [`native.icon`](#native-icon).                                  |
+| `image`       | Optional | Object                                | A picture that is associated with the brand, or grabs the user's attention.  For properties, see [`native.image`](#native-image). |
+| `clickUrl`    | Optional | Object                                | Where the user will end up if they click the ad.  For properties, see [`native.clickUrl`](#native-clickurl).                      |
+| `cta`         | Optional | Object                                | *Call to Action* text, e.g., "Click here for more information".  For properties, see [`native.cta`](#native-cta).                 |
+
+<div class="alert alert-warning">
+  <strong>Prebid Native Ad Validation</strong>
+  <p>
+   Prebid.js validates the assets on native bid responses like so:
+  <ul>
+      <li>
+       If the asset is marked as "required", it checks the bid response to ensure the asset is part of the response
+      </li>
+      <li>
+       However, Prebid.js does not do any additional checking of a required asset beyond ensuring that it's included in the response; for example, it doesn't validate that the asset has a certain length or file size, just that that key exists in the response JSON
+      </li>
+    </ul>
+  </p>
+</div>
+
+<a name="native-image" />
+
+##### `adUnit.mediaTypes.native.image`
+
+{: .table .table-bordered .table-striped }
+| Name            | Scope    | Type                                  | Description                                                                                                                         |
+|-----------------+----------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------|
+| `required`      | Required | Boolean                               | Whether this asset is required.                                                                                                     |
```

Comment:
Is it true that the "required" property is required? (Same comment for all other asset objects below as well).

Reply:
> Is it true that the "required" property is required?

Looks like the answer is no, Prebid.js just filters the list to only required objects according to my reading of https://github.com/prebid/Prebid.js/blob/master/src/native.js#L105

Updated all mentions of the `required` property to note they are optional in dbc79b2